crypto/rsa/rsa_ossl.c

```diff
@@ -859,10 +882,18 @@ static int rsa_ossl_mod_exp(BIGNUM *r0, const BIGNUM *I, RSA *rsa, BN_CTX *ctx)
         BN_free(pr2);
     }
 
+ tail:
     if (rsa->e && rsa->n) {
-        if (!rsa->meth->bn_mod_exp(vrfy, r0, rsa->e, rsa->n, ctx,
-                                   rsa->_method_mod_n))
-            goto err;
+        if (rsa->meth->bn_mod_exp == BN_mod_exp_mont) {
+            if (!BN_mod_exp_mont_consttime(vrfy, r0, rsa->e, rsa->n, ctx,
+                                           rsa->_method_mod_n))
```

Comment:
Wow! This costs 14% in performance. My thought that it would if(BN_FLG_CONSTTIME-here-n-there) would drive it to BN_mod_exp_mont_consttime, but apparently not. Calling BN_mod_exp_mont (which can handle fixed-top)...